Source/WebCore/rendering/RenderLayerCompositor.cpp

```diff
@@ -957,7 +957,7 @@ bool RenderLayerCompositor::updateCompositingLayers(CompositingUpdateType update
 
     if (updateRoot->hasDescendantNeedingUpdateBackingOrHierarchyTraversal() || updateRoot->needsUpdateBackingOrHierarchyTraversal()) {
         ASSERT(m_layersWithUnresolvedRelations.isEmptyIgnoringNullReferences());
-        ScrollingTreeState scrollingTreeState = { 0, 0 };
+        ScrollingTreeState scrollingTreeState = { ObjectIdentifier<ScrollingNodeIDType>(0), 0 };
```

Comment:
This first argument should be `std::nullopt`

Reply:
Fixed.